docs/assets/funfacts.txt

```diff
@@ -10,22 +11,23 @@ At one point, Feather Client was willing to pay $300 USD to add Skytils.
 SkyblockPersonalized was once the cause of many rickrolls from right-clicking\nany of the Mathematics Farming Hoes. (It's been removed by now, of course.)
 The mod NEU has over 80k lines of code, making it\none of the largest mods out there for Skyblock.
 The icon you see in SkyClient's main menu and other places\nis made by none other than the esteemed Zetvue#9664.
-Most of SkyClient's staff have not touched Hypixel Skyblock for several months.
+Most of SkyClient's staff have not touched Hypixel Skyblock for several months. Quit today!
 Grass is a plant with narrow leaves growing from the base. [...]\nGrasses are monocotyledon, herbaceous plants.
 SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-based version, and a web-based version [WIP].
 You'll never believe this, but our Discord's support channels\nare in a category called SUPPORT IS HERE! Use them as necessary.
-SBE has skidded from countless mods including DSM, Skytils and NEU!
 SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
 SkyClient is for more than Skyblock, even though it started out that way.
-SkyClient is a zero profit organization; we have 40 unpaid volunteers around the globe.
+SkyClient is a zero profit organization; we have unpaid volunteers around the globe.
 Most mods' settings are located in a command you can run when in game;\neg Patcher is configurable with `/patcher`.
 At one point in Minecraft, there was a rare chance of Snow Golems naturally spawning.
 When you're asked to update your mods, you can click on a mod's name to stop it\nfrom being replaced with SkyClient's latest version.
 W[h]yvest
+Quagsire
+Boost our discord!\n(https://inv.wtf/skyclient)
+RUMOUR. Open your ears; 9r"5j5&?OWTY Z0d
```

Comment:
is this a reference to https://books.google.com/books?id=bZcKSdKK9NYC&pg=PA9&lpg=PA9&dq=open+your+ears+OWTY+Z0d&source=bl&ots=p9pzXPN4Dk&sig=ACfU3U2G05Kfgy45JEcxBZctRU-HE5OWcg&hl=en&sa=X&ved=2ahUKEwiOnsLOi6f6AhWKHjQIHRuTBpkQ6AF6BAgWEAM#v=onepage&q=open%20your%20ears%20OWTY%20Z0d&f=false

Reply:
bro what

Reply:
what does this come from

Reply:
you will never know